python/dlisio/plumbing/errorhandler.py

```diff
@@ -4,10 +4,124 @@
 from .. import core
 
 class ErrorHandler(core.error_handler):
-    """
-    TODO ErrorHandler
-    """
+    """ Defines rules about error handling
+
+    Many .dlis files happen to be not compliant with specification or simply
+    broken. This class gives user some control over handling of such files.
+
+    For example:
+
+    * Users can throw on specification violations which dlisio doesn't consider
+      critical.
+    * Users can escape errors if they want to read as much data as possible.
+
+    Error handler does not cover all the warnings and exceptions.
+    It is mostly intended to be used for data processing, not for data
```

Comment:
Im no huge fan of the word processing. Everywhere else dlisio is categorised as a parser or dlis-reader. Now suddenly its a processor? Please don't confuse people with new words if not necessary. (read or parse are not inherently better than processing, but we don't need yet another pseudo word in the mix)